engine/src/test/java/org/hibernate/validator/test/internal/constraintvalidators/hv/ScriptAssertValidatorTest.java

```diff
@@ -113,6 +118,35 @@ public void scriptExpressionReturningNoBooleanRaisesException() throws Exception
 		validator.isValid( new Object(), null );
 	}
 
+	@Test
+	public void reportOnField() {
```

Comment:
Please add the `@TestForIssue` annotation. It's less important than for bugs but it's a good habit.